<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportWindow</class>
 <widget class="QWidget" name="ImportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Import Run</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>32</y>
     <width>291</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Miles</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Route Name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Elevation Gain</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Max Heart Rate</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Avg. Heart Rate</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Fastest Mile</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Calories</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Tags</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_Tags"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_Cals"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_Fastest"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_AVR"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_MHR"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_EGain"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_Time"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_Miles"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_Route"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_Import">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
